django/core/management/commands/listurls.py

```diff
@@ -0,0 +1,277 @@
+import json
+from importlib import import_module
+from io import StringIO
+
+from django.conf import settings
+from django.contrib.admindocs.views import (
+    extract_views_from_urlpatterns,
+    simplify_regex,
+)
+from django.core.management import color
+from django.core.management.base import BaseCommand, CommandError, CommandParser
+from django.utils import termcolors
+
+FORMATS = (
+    "aligned",
+    "table",
+    "verbose",
+    "json",
+    "pretty-json",
+)
+
+
+COLORLESS_FORMATS = (
+    "json",
+    "pretty-json",
+)
+
+
+class Command(BaseCommand):
+    help = "List URL patterns in the project with optional filtering by prefixes."
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        self.style.HEADER = termcolors.make_style(opts=("bold",))
+        self.style.ROUTE = termcolors.make_style(opts=("bold",))
+        self.style.VIEW_NAME = termcolors.make_style(fg="yellow", opts=("bold",))
+        self.style.NAME = termcolors.make_style(fg="red", opts=("bold",))
```

Comment:
```suggestion
        self.style.HIGHLIGHT = termcolors.make_style(fg="yellow", opts=("bold",))
```
I think red is associated with errors and so I would change this to be a highlight style.
Part of the reason I think the different colors existed before was around the visual difference due to the dense format, without that format I think 1 color is enough